### DISCUSSION TITLE
How to set 'Debug=True' in '.env' when i test on local host

### DISCUSSION THREAD

**Discussion by cha2hyun** [vercel/next.js#45029] [Help] [answered]
## Problem
`.env` has `const Debug=True`. 
I manually change the debug value when testing on localhost.
The problem is that sometimes I forgot to change the debug value to False when deploying to the server.

## Problem
So I would like to know how to change `Debug=True` in `.env` automatically when starting with `yarn dev`. 
or some tricks to make it debug mode in localhost only

Any Help!??

**Comment by icyJoseph** [comment, marked as answer]
Start your app with `DEBUG=True next dev`, in your package json:

```json
"scripts": {
    "dev": "DEBUG=True next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "export": "next export"
  },
```

What's even funnier is that you could just check `process.env.NODE_ENV === "development"`, which is already provided by the framework.

**Reply by cha2hyun** [reply]
Thanks for tips !
